bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -276,18 +297,23 @@ def submit_otp(self, otp):
+                proceed_btn = self.get_element(
+                    "proceedBtn", "id", timeout=8, throw="ignore"
+                )
+                if proceed_btn:
+                    proceed_btn.click()
+                    # proceed btn prompt comes twice in the UI
+                    proceed_btn = self.get_element(
+                        "proceedBtn", "id", timeout=8, throw="ignore"
+                    )
+                    proceed_btn.click()
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)